openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -3,14 +3,16 @@
 <ol class="submission__answer__display__content">
     {% for part in answer.parts %}
         <li class="submission__answer__part">
+            <div class="submission__answer__part__text">
+                <h5 class="submission__answer__part__text__title">{% trans "The question for this section" %}</h5>
+            </div>
```

Comment:
Note: This was moved and the class changed so that it would not be shown to the user.  This was per A11y review to improve Header levels and screen reader flow.